lib/src/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:async';
+import 'dart:math';
+import '../styles/constants.dart';
+import 'i_dice_bloc.dart';
+
+class DiceBloc extends IDiceBloc {
+  List<int> _diceValues = [];
+  StreamController<List<int>> _streamValueController =
+      StreamController<List<int>>.broadcast();
+  StreamController<String> _streamDicesScoreController =
+      StreamController<String>.broadcast();
+
+  @override
+  void dispose() {
+    _streamValueController.close();
+    _streamDicesScoreController.close();
+  }
+
+  @override
+  void initialize() {}
+
+  @override
+  void roll() {
+    this._diceValues.clear();
+    int score = 0;
```

Comment:
score is a private var of roll function, you need to add '_' at the beginning of the variable

Reply:
done